### ISSUE TITLE
EACCES: permission denied, open '/private/var/db/Accessibility/.VoiceOverAppleScriptEnabled'

### ISSUE THREAD

**Issue by diegohaz** [guidepup/setup#23]
Recently, I've been seeing an error while running `guidepup/setup-action` on macOS 12.6.9:

```
[!] Error: Unable to write to the VoiceOver database file - SIP might not be disabled

EACCES: permission denied, open '/private/var/db/Accessibility/.VoiceOverAppleScriptEnabled'

Unable to complete environment setup

Please raise new issues at: https://github.com/guidepup/setup/issues
```

Example: https://github.com/ariakit/ariakit/actions/runs/6818966513/job/18545531840

**Comment by cmorten** [comment]
I’ve been seeing an increase in this lately as well. I suspect we will need to raise an issue against https://github.com/actions/runner-images/issues to triage.

It seems it is sometime passing, mostly failing, which makes me wonder if there has been a bad build of agents, and depending on which you get it will work or not.

Some references of original work to enable this:

- https://github.com/actions/runner-images/issues/4770
- https://github.com/actions/runner-images/pull/4824

**Comment by cmorten** [comment]
This was the spec from your failing build:

```
Current runner version: '2.311.0'
Operating System
  macOS
  12.6.9
  21G726
Runner Image
  Image: macos-12
  Version: 20230921.1
  Included Software: https://github.com/actions/runner-images/blob/macOS-12/20230921.1/images/macos/macos-12-Readme.md
  Image Release: https://github.com/actions/runner-images/releases/tag/macOS-12%2F20230921.1
Runner Image Provisioner
  2.0.312.1
```

**Comment by cmorten** [comment]
And an example spec from a passing build 3 weeks ago:

```
Current runner version: '2.310.2'
Operating System
  macOS
  12.7
  21G816
Runner Image
  Image: macos-12
  Version: 20230921.4
  Included Software: https://github.com/actions/runner-images/blob/macOS-12/20230921.4/images/macos/macos-12-Readme.md
  Image Release: https://github.com/actions/runner-images/releases/tag/macOS-12%2F20230921.4
Runner Image Provisioner
  2.0.312.1
```

REF: https://github.com/guidepup/guidepup/actions/runs/6585897797/job/17893182985

**Comment by jlp-craigmorten** [comment]
I've released a new patch version of this package and the GH action, would be good to know if it improves matters!

**Comment by diegohaz** [comment]
@jlp-craigmorten Looks like it's working now. Thank you so much!
